server/fixity.go

```diff
@@ -18,6 +32,8 @@ type FixityDB interface {
 	// the empty string is returned. It will also not return anything in an
 	// "error" state.
 	NextFixity(cutoff time.Time) string
+	GetFixityById(id string) *fixity
+	GetFixity(start string, end string, item string, status string) []*fixity
```

Comment:
why not make start and end of type `time.Time`. With the zero time meaning the interval is open-ended. Then the sql handlers can format the time as they see fit.

Reply:
I'm sticking with the RFC339 string. Our beloved QL library stores the time as a btye array, and one must undergo chicanery to use time.Time()

Reply:
I disagree...here is an example of using `time.Time` with the QL library: https://github.com/ndlib/bendo/blob/master/server/db_ql.go#L87-L102
and here: https://github.com/ndlib/bendo/blob/master/server/db_ql.go#L188-L196

Anyway, the QL library is only used for development...how much of the design should it dictate?